### PR TITLE
Add new package: slider

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/slider/package.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    version('0.92.0', sha256='485f02f4f9f0b270017717c9471b83b0d77d005d25261b741fb381791ce838b9')
+    version('0.91.0', sha256='212a5cde6de60060c9a081f553d66940b70af4bccb469072febb554c4005bcef')
+    version('0.90.2', sha256='410941f772d29f564c4bb90ca0631f29dc895f509048cb6052f8695302e3f944')
+
+    depends_on('maven', type='build')
+    depends_on('java@8', type=('build', 'run'))
+    depends_on('python@2.7.0:2.7.99', type='run')
+
+    def url_for_version(self, version):
+        ver = version.up_to(3)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/slider/package.py]: It doesn't seem like this line is necessary

File: var/spack/repos/builtin/packages/slider/package.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        return "http://archive.apache.org/dist/incubator/slider/{0}-incubating/apache-slider-{0}-incubating-source-release.tar.gz".format(ver)
+
+    def install(self, spec, prefix):
+        ver = self.version.up_to(3)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/slider/package.py]: Same here

File: var/spack/repos/builtin/packages/slider/package.py (hunk 1 above)
Comment (scheibelp) [reply var/spack/repos/builtin/packages/slider/package.py]: There are a couple examples of RC0/RC1 sub-releases (e.g. `0.90.2-RC1`), although it appears that in these cases the `RC*` component is included in the URL name but not in this function.

File: var/spack/repos/builtin/packages/slider/package.py (hunk 2 above)
Comment (darmac) [reply var/spack/repos/builtin/packages/slider/package.py]: Fixed in https://github.com/spack/spack/pull/17799/commits/a1ad5a98d0dcf4f54c5dff12c51a6f98e4e2f63b

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**